erigon-lib/state/aggregator.go

```diff
@@ -1250,7 +1250,7 @@ func (r RangesV3) String() string {
 	aggStep := r.domain[kv.AccountsDomain].aggStep
 	for p, mr := range r.invertedIndex {
 		if mr != nil && mr.needMerge {
-			ss = append(ss, mr.String(string(p), aggStep))
```

Comment:
RangedV3#String() suffers as we don't have kv.INvertedIndex info here

Reply:
let's add this info - because it's likely useful for debugging and printing